src/attr/_make.py

```diff
@@ -1672,6 +1672,36 @@ def fields_dict(cls):
     return ordered_dict(((a.name, a) for a in attrs))
 
 
+def resolve_types(cls, globalns=None, localns=None):
+    """
+    Resolve any strings and forward annotations in type annotations.
```

Comment:
Could you add a brief summary why one would want to specify *globalns* or *localns* please?